apisix/plugins/consumer-restriction.lua

```diff
@@ -53,6 +53,38 @@ local schema = {
                 rejected_code = {type = "integer", minimum = 200, default = 403}
             },
             required = {"whitelist"},
+        },
+        {
+            title = "allowed_methods",
```

Comment:
Would it be better to use `allowed_by_methods`?

Reply:
agree